x-pack/plugins/timelines/public/hooks/use_status_bulk_action_items.tsx

```diff
@@ -28,7 +29,7 @@ export const useStatusBulkActionItems = ({
   onUpdateFailure,
   timelineId,
 }: StatusBulkActionsProps) => {
-  const { updateAlertStatus } = useUpdateAlertsStatus(timelineId != null);
+  const { updateAlertStatus } = useUpdateAlertsStatus(timelineId !== STANDALONE_ID);
```

Comment:
I think just passing in the `timelineId` into the `useUpdateAlertsStatus` hook is cleaner than continuing with this conditional assertion in the parameters which then gets passed into the hook.

It'd be clearer and safer IMO if the hook itself determined when to use the signals route vs the RAC route. That logic should be encapsulated in the hook itself, not outside of it.

I think we should update the signature of the `useUpdateAlertsStatus` to accept the timelineId, not a boolean, and then update the conditional checking inside of 

https://github.com/elastic/kibana/blob/c365e608192bb747b2378d9864358d7c36eec2b2/x-pack/plugins/timelines/public/container/use_update_alerts.ts#L42

to become

```
if (timelineId !== STANDALONE_ID) {...
```

I feel this will reduce the likelihood of bugs moving forward.

Reply:
Ok we discuss it and we agree to keep it the way it is